domain/src/main/java/org/oppia/android/domain/state/BUILD.bazel

```diff
@@ -0,0 +1,57 @@
+"""
+build bazel for domain state
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
+    name = "state_deck",
+    srcs = [
+        "StateDeck.kt",
+    ],
+    visibility = ["//:oppia_api_visibility"],
+    deps = [
+        "//model:exploration_checkpoint_java_proto_lite",
+        "//model:exploration_java_proto_lite",
+        "//model:feedback_reporting_java_proto_lite",
+        "//model:interaction_object_java_proto_lite",
+        "//model:onboarding_java_proto_lite",
+        "//model:question_java_proto_lite",
+        "//model:subtitled_html_java_proto_lite",
+        "//model:topic_java_proto_lite",
```

Comment:
We shouldn't be adding all of these--you should look at the import statements within StateDeck to see which model dependencies are needed, then add them here & build locally to verify.

Ditto for the other new libraries below.